### PR TITLE
feat(MangaFire): add presence

### PR DESCRIPTION
**Pull request by Extiriority**
## Description 
A rich presence addon for MangaFire, A free manga platform where you can read different manga from the makers of 9anime.

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</a>
</a>
</a>
</a>
</a>
</a>

</details>

### CODE REVIEW

File: websites/M/MangaFire/presence.ts [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+			pathArr[1],
+			pathArr[2],
+			document.querySelector(".name")?.textContent
+		),
+		presenceData: PresenceData = {
+			largeImageKey: largeImageKey || Assets.Logo,
+			startTimestamp: browsingTimestamp,
+			details,
+		},
+		[showDetail] = await Promise.all([presence.getSetting<boolean>("details")]);
```
Comment (Bas950) [inline websites/M/MangaFire/presence.ts:23]: Promise all ain't needed for 1 promise

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/M/MangaFire/presence.ts (hunk 1 above)
Comment (Extiriority) [reply websites/M/MangaFire/presence.ts]: Ah, my mistake. I had another setting but deleted it and forgot to change it. thx

File: websites/M/MangaFire/presence.ts [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+		presenceData: PresenceData = {
+			largeImageKey: largeImageKey || Assets.Logo,
+			startTimestamp: browsingTimestamp,
+			details,
+		};
+	if (smallImageKey) presenceData.smallImageKey = smallImageKey;
+	if (state) presenceData.state = state;
+
+	if (
+		![await presence.getSetting<boolean>("details")][0] &&
```
Comment (Bas950) [inline websites/M/MangaFire/presence.ts:27]:
```suggestion
		!(await presence.getSetting<boolean>("details")) &&
```

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]

File: websites/M/MangaFire/metadata.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+		"name": "Extiriority"
+	},
+	"service": "MangaFire",
+	"description": {
+		"en": "MangaFire.to is a completely free online modern manga platform with zero pop-up ads where we offer an extensive collection of Manga, Light-Novel, Manhwa and many more to read.",
+		"nl": "MangaFire.to is een compleet gratis online moderne manga platform zonder pop-up advertentie waar wij een groot collectie aanbieden zoals Manga, Light-Novel, Manhwa en veel meer om te lezen."
+	},
+	"url": "mangafire.to",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/KXGOlbM.png",
```
Comment (DarkVillager) [inline websites/M/MangaFire/metadata.json:14]: Low quality...
```suggestion
	"logo": "https://i.imgur.com/356Spon.png",
```

File: websites/M/MangaFire/presence.ts [hunk 4]
```diff
@@ -0,0 +1,121 @@
+const presence = new Presence({ clientId: "1080885668248559648" }),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+enum Assets {
+	Logo = "https://i.imgur.com/KXGOlbM.png",
```
Comment (DarkVillager) [inline websites/M/MangaFire/presence.ts:5]: Low quality
```suggestion
	Logo = "https://i.imgur.com/356Spon.png",
```

**Review by DarkVillager — CHANGES_REQUESTED** [review]

File: websites/M/MangaFire/metadata.json [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+	"author": {
+		"id": "135058085386387457",
+		"name": "Extiriority"
+	},
+	"service": "MangaFire",
+	"description": {
+		"en": "MangaFire.to is a completely free online modern manga platform with zero pop-up ads where we offer an extensive collection of Manga, Light-Novel, Manhwa and many more to read.",
+		"nl": "MangaFire.to is een compleet gratis online moderne manga platform zonder pop-up advertentie waar wij een groot collectie aanbieden zoals Manga, Light-Novel, Manhwa en veel meer om te lezen."
+	},
+	"url": "mangafire.to",
```
Comment (theusaf) [inline websites/M/MangaFire/metadata.json:12]: Website uses `www` subdomain.
```suggestion
	"url": [
		"mangafire.to",
		"www.mangafire.to"
	],
```

**Review by theusaf — CHANGES_REQUESTED** [review]

**Review by Timeraa — APPROVED** [review]

**Pull request merged**